examples/HelloWorldEclipse/src/org/androidannotations/helloworldeclipse/ActivityWithExtra.java

```diff
@@ -1,42 +0,0 @@
-package org.androidannotations.helloworldeclipse;
```

Comment:
The Eclipse example contained much more example code than the Gradle or Maven one. Can you copy all the example codes to both samples?